### ISSUE TITLE
Improve the regression queue

### ISSUE THREAD

**Issue by rtholmes** [ubccpsc310/classy#27]
When the standard queue is empty and nothing is waiting, but there are jobs waiting on the regression queue, service the regression items.

For safety, do not use the express queue for these jobs.

The whole point is to improve this kind of situation:

```
AutoTest::tick(..) - start; standard - #wait: 0, #run: 0; express - #wait: 0, #run: 0; regression - #wait: 154, #run: 1; schedule - #wait: 0.
```

While this won't do much during the day when the queues are being kept busy by students working, overnight and in the morning (and right after deadlines) it can help drain the queue much more quickly.

**Comment by rtholmes** [comment]
fixed in #4aea358

**Comment by rtholmes** [comment]
Deployed and working.
